nomad/worker.go

```diff
@@ -134,8 +134,9 @@ func (w *Worker) run() {
 func (w *Worker) dequeueEvaluation(timeout time.Duration) (*structs.Evaluation, string, bool) {
 	// Setup the request
 	req := structs.EvalDequeueRequest{
```

Comment:
So I was thinking we want it to spam the logs so the operator knows they are doing something bad. I agree that we should detect and start tuning back tho. But I don't think we should do anything permanent. Thoughts?